From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Sultan Alsawaf <sultan@osomprivacy.com>
Date: Tue, 11 Apr 2023 11:22:49 -0700
Subject: [PATCH] msm: ipa3: Silence PM notifier log spam

We don't need to know that PM notifiers are working. Silence the entry/exit
log spam in ipa_pm_notify().

Change-Id: I9bae89fa5e27b8835683ec1423ecc751491ea13c
Signed-off-by: Sultan Alsawaf <sultan@osomprivacy.com>
Signed-off-by: Sevenrock <sevenrock@hotmail.de>
---
 techpack/dataipa/drivers/platform/msm/ipa/ipa_v3/ipa.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/techpack/dataipa/drivers/platform/msm/ipa/ipa_v3/ipa.c b/techpack/dataipa/drivers/platform/msm/ipa/ipa_v3/ipa.c
index 2fe49feaa860..f8c211adb817 100644
--- a/techpack/dataipa/drivers/platform/msm/ipa/ipa_v3/ipa.c
+++ b/techpack/dataipa/drivers/platform/msm/ipa/ipa_v3/ipa.c
@@ -456,7 +456,7 @@ EXPORT_SYMBOL(ipa_smmu_free_sgt);
 
 static int ipa_pm_notify(struct notifier_block *b, unsigned long event, void *p)
 {
-	IPAERR("Entry\n");
+	IPADBG("Entry\n");
 	switch (event) {
 		case PM_POST_SUSPEND:
 #ifdef CONFIG_DEEPSLEEP
@@ -468,7 +468,7 @@ static int ipa_pm_notify(struct notifier_block *b, unsigned long event, void *p)
 #endif
 			break;
 	}
-	IPAERR("Exit\n");
+	IPADBG("Exit\n");
 	return NOTIFY_DONE;
 }
 
-- 
2.39.5

